src/Uno.UI/UI/Xaml/UIElement.Pointers.wasm.cs

```diff
@@ -281,31 +281,39 @@ private enum HitTestVisibility
 		/// <returns></returns>
 		private static object CoerceHitTestVisibility(DependencyObject dependencyObject, object baseValue)
 		{
-			var element = (UIElement)dependencyObject;
 
 			// The HitTestVisibilityProperty is never set directly. This means that baseValue is always the result of the parent's CoerceHitTestVisibility.
-			var baseHitTestVisibility = (HitTestVisibility)baseValue;
-
-			// If the parent is collapsed, we should be collapsed as well. This takes priority over everything else, even if we would be visible otherwise.
-			if (baseHitTestVisibility == HitTestVisibility.Collapsed)
+			var uiElement = (UIElement)dependencyObject;
+			var coercedValue = CoerceInner(uiElement, (HitTestVisibility)baseValue);
+			if (FeatureConfiguration.UIElement.AssignDOMXamlProperties)
 			{
-				return HitTestVisibility.Collapsed;
+				uiElement.UpdateDOMXamlProperty("HitTestVisibility", coercedValue);
 			}
```

Comment:
I would have put this in the `OnHitTestVisibilityChanged`, is there a reason to put it here?

Reply:
I wasn't sure if it would be raised for the element calling `UpdateHitTest()`... but looking more closely, I think it probably will?